tcmur_cmd_handler.c

```diff
@@ -89,9 +89,12 @@ static int check_lba_and_length(struct tcmu_device *dev,
 					   ASC_INTERNAL_TARGET_FAILURE, NULL);
 	}
 
-	if (lba >= num_lbas || lba + sectors > num_lbas)
+	if (lba + sectors > num_lbas || lba + sectors < lba) {
+		tcmu_err("cmd exceeds last lba %"PRIu64" (lba %"PRIu64", xfer len %"PRIu32")\n",
+				num_lbas, lba, sectors);
```

Comment:
Just a dumb nit, but could you follow the coding style in most of the other code?

Line the num_lbas line with the top line's open "(" first char. Look at the iov_length check's tcmu_err above this chunk of code for an example.